lib/options.js

```diff
@@ -259,11 +259,11 @@ function validate(opts, validationOptions) {
   var result = validator.validate(opts, '/Options', validationOptions);
 
   // check empty interfaces
-  if (opts.hasOwnProperty('interfaces')) {
-    if (opts.interfaces.length === 0) {
-      result.addError('no interfaces were defined');
-    }
-  }
+  // if (opts.hasOwnProperty('interfaces')) {
+  //   if (opts.interfaces.length === 0) {
+  //     result.addError('no interfaces were defined');
+  //   }
+  // }
```

Comment:
why this has been commented? If this is not needed anymore, we should just remove it.